spacing/_units.scss

```
@@ -0,0 +1,20 @@
+// Design System Spacing Units
+
+:root {
+  // Base spacing units
```

Do we miss this value?
```suggestion
--unit-0: 0px;
```